### ISSUE TITLE
[a11y] Firefox /enterprise page accessibility improvements

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15341]
# Axe-core® Accessibility Results for /en-US/firefox/enterprise/

Page URL: https://www-dev.allizom.org/en-US/firefox/enterprise/

## axe-core found 1 violation

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | `<ul>` and `<ol>` must only directly contain `<li>`, `<script>` or `<template>` elements | list | WCAG 2 Level A, WCAG 1.3.1 | serious | 1   |

### Failed

1. `<ul>` and `<ol>` must only directly contain `<li>`, `<script>` or `<template>` elements

[Learn more](https://dequeuniversity.com/rules/axe/4.10/list?application=playwright)

list

WCAG 2 Level A, WCAG 1.3.1

Ensure that lists are structured correctly

serious

Issue Tags: cat.structure wcag2a wcag131 EN-301-549 EN-9.1.3.1

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.mzp-l-columns<br>```<br><br>**Element source**<br><br>```<br><ul class="mzp-l-columns mzp-t-columns-three"><br>``` | Fix all of the following:<br><br>*   List element has direct children that are not allowed: div |

**Comment by alexgibson** [comment]
Tip: the three picto components contained within the `<ul>` should be `<li>` elements instead of generic `<div>` elements.

**Comment by clarapenn** [comment]
Hi, I'd like to work on this, please. Thanks!
